Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang:stretch as builder
+
+RUN mkdir -p $GOPATH/pkg/mod $GOPATH/bin
+
+RUN apt-get update \
+  && apt-get install -y ruby ruby-dev curl  \
+  && gem install --no-document fpm \
```

Comment:
You can remove this line and `ruby ruby-dev` - none of this is required. I've tested it without fpm. Fpm is used to build rpm or deb packages.

Reply:
was updated and removed in last commit